components/ip2proxy/actions/query-ip-info/query-ip-info.mjs

```diff
@@ -0,0 +1,43 @@
+import { axios } from "@pipedream/platform";
```

Comment:
Essentially the same comments I made for `ip2location/actions/query-ip-info/query-ip-info.mjs` applies to this file